src/devices/candeo.ts

```diff
@@ -319,4 +319,53 @@ export const definitions: DefinitionWithExtend[] = [
             await endpoint1.read("genBasic", [switchTypeAttribute], {manufacturerCode: manufacturerSpecificClusterCode});
         },
     },
+    {
+        fingerprint: [{modelID: "C-ZB-SEWA", manufacturerName: "Candeo"}],
+        model: "C-ZB-SEWA",
+        vendor: "Candeo",
+        description: "Candeo C-ZB-SEWA Water Sensor",
+        extend: [m.battery()],
+        fromZigbee: [fz.ias_enroll, fz.ias_water_leak_alarm_1, fz.ias_water_leak_alarm_1_report],
+        exposes: [e.water_leak()],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ["ssIasZone"]);
+            await endpoint.read("ssIasZone", ["zoneStatus"]);
+        },
+    },
+    {
+        fingerprint: [{modelID: "C-ZB-SETE", manufacturerName: "Candeo"}],
+        model: "C-ZB-SETE",
+        vendor: "Candeo",
+        description: "Temperature & humidity sensor",
+        extend: [m.temperature(), m.humidity(), m.battery()],
+    },
+    {
+        fingerprint: [{modelID: "C-ZB-SEDC", manufacturerName: "Candeo"}],
+        model: "C-ZB-SEDC",
+        vendor: "Candeo",
+        description: "Door contact sensor",
+        extend: [m.battery()],
+        fromZigbee: [fz.ias_contact_alarm_1, fz.ias_contact_alarm_1_report],
+        exposes: [e.contact()],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ["ssIasZone"]);
+            await endpoint.read("ssIasZone", ["zoneStatus"]);
+        },
+    },
+    {
+        fingerprint: [{modelID: "C-ZB-SEMO", manufacturerName: "Candeo"}],
+        model: "C-ZB-SEMO",
+        vendor: "Candeo",
+        description: "Motion sensor",
+        extend: [m.battery(), m.illuminance({reporting: {min: 1, max: 65535, change: 1}})],
```

Comment:
```suggestion
        extend: [m.battery(), m.illuminance({reporting: {min: 1, max: 65535, change: 1}}), m.iasZoneAlarm({zoneType: "occupancy", zoneAttributes: ["alarm_1"]})],
```

This allows you to get red of all fromZigbee/exposes and configure. Please also do it for the other devices